js/src/mobx/hardwareStore.js

```diff
@@ -147,6 +164,19 @@ export default class HardwareStore {
     return this._ledger.signTransaction(transaction);
   }
 
+  pinMatrixAck (device, passcode) {
+    return this._api.parity
+      .trezor('pin_matrix_ack', device.path, passcode)
+      .then((message) => {
+        this.scan();
+        // true/false is returned
+        return JSON.parse(message);
+      })
+      .catch((err) => {
+        return err;
```

Comment:
This converts an error into a positive response, was that intended? Is the calling code aware of that?